### ISSUE TITLE
Incorrect detection of Apache 2.0 license in 'cordova-plugin-keyboard'

### ISSUE THREAD

**Issue by zackypick** [davglass/license-checker#224]
license-checker erroneously reports license for npm package _cordova-plugin-keyboard@1.2.0_ - here is the resulting package object from license-checker (note the _licenses_ field):

```
{
  licenses: 'Custom: https://github.com/apache/cordova-plugins/tree/master/keyboard',
  repository: 'https://github.com/cjpearson/cordova-plugin-keyboard',
  publisher: 'Apache Software Foundation',
  path: 'c:\\projects\\c-b4-mobile\\dashboard@2.0\\node_modules\\cordova-plugin-keyboard',
  licenseFile: 'c:\\projects\\c-b4-mobile\\dashboard@2.0\\node_modules\\cordova-plugin-keyboard\\README.md'
}
```

While in fact _cordova-plugin-keyboard@1.2.0_ package.json file has the license field set properly:

```
...
   "license": "Apache 2.0",
...
```

Note, that when manually updating the package's license field (under node_modules) to _Apache 2.0_, then license-checker indeed reports the right license:

```
{
  licenses: 'Apache-2.0',
  repository: 'https://github.com/cjpearson/cordova-plugin-keyboard',
  publisher: 'Apache Software Foundation',
  path: 'c:\\projects\\c-b4-mobile\\dashboard@2.0\\node_modules\\cordova-plugin-keyboard',
  licenseFile: 'c:\\projects\\c-b4-mobile\\dashboard@2.0\\node_modules\\cordova-plugin-keyboard\\README.md'
}
```
